031-DevOpsWithGitHub/readme.md

```diff
@@ -0,0 +1,53 @@
+# What The Hack - DevOps with GitHub
+
+## Introduction
+DevOps is a journey not a destination. Our goal when writing this challenged based hack is to introduce you to GitHub and some common DevOps practices. We also understand that your choice of programming language and DevOps processes might differ from the ones we will be using in this hack, that is OK. Our intent was to select some of the most common programming languages and highlight industry best practices, with an emphasis on showing how GitHub can help you on your DevOps journey, so that you can apply this in your environment with the languages and tools that you use.
+
+## Learning Objectives
+
+This DevOps with GitHub hack will help you learn:
+
+1. How to use GitHub to manage source control
+1. How to use GitHub for Project Management
+1. How to use GitHub Actions for CI & CD
+1. Monitoring apps with Application Insights
+
+## Challenges
+ - [Challenge 0](./Student/challenge00.md) - Setup and Introduction
+ - [Challenge 1](./Student/challenge01.md) - Track your work with GitHub Project Boards
+ - [Challenge 2](./Student/challenge02.md) - Centralize your code with GitHub Repos
+ - [Challenge 3](./Student/challenge03.md) - Infrastructure as Code
+ - [Challenge 4](./Student/challenge04.md) - Continuous Integration
+ - [Challenge 5](./Student/challenge04.md) - Build and push Docker image to container registry
```

Comment:
Challenges 5 through 9 link to challenge N-1 ;)  Please fix!

Reply:
Thanks...fixed!